Kitodo/src/main/java/org/kitodo/services/data/BatchService.java

```diff
@@ -281,10 +281,12 @@ public void addAllObjectsToIndex() throws CustomResponseException, InterruptedEx
     @Override
     public BatchDTO convertJSONObjectToDTO(JSONObject jsonObject, boolean related) throws DataException {
         BatchDTO batchDTO = new BatchDTO();
-        batchDTO.setTitle(getStringPropertyForDTO(jsonObject, "title"));
-        batchDTO.setType(getStringPropertyForDTO(jsonObject, "type"));
+        batchDTO.setId(getIdFromJSONObject(jsonObject));
+        JSONObject batchObject = getSource(jsonObject);
+        batchDTO.setTitle(getStringPropertyForDTO(batchObject, "title"));
+        batchDTO.setType(getStringPropertyForDTO(batchObject, "type"));
         if (!related) {
-            batchDTO = convertRelatedJSONObjects(jsonObject, batchDTO);
+            batchDTO = convertRelatedJSONObjects(batchObject, batchDTO);
```

Comment:
batchObject sounds too much like it's a bean.
I would prefere batchJSONObject

Reply:
renamed